jbmc/regression/jbmc-strings/ConstantEvaluationContains/notcontains.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+Main.class
+--function Main.notContains --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^Generated [1-9]\d* VCC\(s\), [1-9]\d* remaining after simplification$
```

Comment:
As a non-noprop case I expect these *should* be decided by the simplifier

Reply:
These cases usually resolve in a VCC but with a directly false guard somewhere. Might be to do with building a trace? Always been like this when I've tested and just assumed that's how things were.